crates/p2p_proto/proto/sync.proto

```diff
@@ -80,3 +72,13 @@ message Status {
   starknet.common.FieldElement hash = 2;
   starknet.common.FieldElement chain_id = 3;
 }
+
+message GetContractClasses {
+  repeated starknet.common.FieldElement class_hashes = 1;
+  uint64 count = 2;
+  uint64 size_limit = 3;
```

Comment:
Similarly, what's the semantic meaning of `size_limit`? And what's the unit we're measuring size in?

Reply:
It was supposed to follow the convention in other requests but those don't really specify if it's the entire message size in bytes or whatever else this could be...

Reply:
I'd keep the size_limit(s) in all of the requests and let us specify the concrete the semantics later on.

Reply:
Absolutely. What I'm missing a bit from the .proto files is maybe a few comments that make the semantic meaning clear for these fields. We can add these later of course.